clients/ctl/admin-ui/cypress/e2e/systems.cy.ts

```diff
@@ -44,62 +44,171 @@ describe("System management page", () => {
   });
 
   describe("Can create a new system", () => {
-    it("Can create a system via yaml", () => {
+    beforeEach(() => {
```

Comment:
this diff is lousy, but basically I put the yaml tests in one describe block so the manual tests could go in a separate describe block. no changes to the yaml tests. just one huge test for the manual one.

Reply:
"Hide whitespace changes" is our friend

![Screen Shot 2022-09-15 at 11 55 44](https://user-images.githubusercontent.com/2236777/190486639-3b34b10d-557b-41cc-ad87-1ddb899a7a8a.png)